docker/Dockerfile

```diff
@@ -164,6 +164,10 @@ COPY firefly/docker/local.xml conf/Catalina/localhost
 #copy all wars, typically there should only be one
 COPY --from=builder /opt/work/${build_dir}/build/dist/*.war ${CATALINA_HOME}/webapps-ref/
 
+# Enable GZIP compression in Tomcat's HTTP connector
+RUN sed -i 's/<Connector port="8080" protocol="HTTP\/1.1"/<Connector port="8080" protocol="HTTP\/1.1" compression="on" useSendfile="false" compressibleMimeType="text\/html,text\/plain,text\/css,text\/javascript,application\/javascript,application\/json,application\/xml,text\/xml,application\/x-votable+xml,application\/x-yaml,application\/ld+json,image\/svg+xml,text\/csv,application\/xhtml+xml,application\/rss+xml,application\/atom+xml,application\/x-font-ttf,font\/otf,font\/woff,font\/woff2"/' \
```

Comment:
Can you make this a script? it is very hard to understand.

Reply:
The changes to developer's setup are here: https://github.com/IPAC-SW/irsa-ife/pull/440/files#diff-865bcc67bc7ab28b0cfc427453a74b51aae44b381a574d6769cf78e1f2cf4c44

I don’t think a script would help, but I can add more comments to explain what it’s doing.